tools/cloud-build/daily-tests/ansible_playbooks/test-validation/test-gke-job.yml

```diff
@@ -20,21 +20,21 @@
 
 - name: Get cluster credentials for kubectl
   delegate_to: localhost
-  ansible.builtin.command: gcloud container clusters get-credentials {{ deployment_name }} --region {{ region }} --project {{ custom_vars.project }}
+  ansible.builtin.command: gcloud container clusters get-credentials {{ deployment_name }} --region {{ region }} --project {{ custom_vars.project }} --verbosity=debug
 
 - name: Execute the job
   delegate_to: localhost
   ansible.builtin.shell: |
     array=({{ workspace }}/{{ deployment_name }}/primary/my-job*)
-    kubectl create -f ${array[0]}
+    kubectl create -f ${array[0]} -v=9
```

Comment:
add a description to the PR explaining why `v=9` was chosen. same goes for `--verobsity=debug`